ml4gw/waveforms/ringdown.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import torch
+
+from ml4gw.types import ScalarTensor
+
+c = 299792458.0  # speed of ligt
+G = 6.67430e-11  # gravitational constant
```

Comment:
Ravi just recently added a [constants.py](https://github.com/ML4GW/ml4gw/blob/dev/ml4gw/constants.py) file that you can import these values from (e.g. `from ..constants import C, G`). For consistency, let's switch to using that file